### PR TITLE
New Relic Synthetics quota

### PR DESCRIPTION
**Pull request by tyherbert**
We are adding a quota for New Relic to 10K per workspace, updating docs to reflect this change.

Closes #

## Summary

**[Doc Page Title](https://docs.pantheon.io/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)

### CODE REVIEW

File: source/content/guides/new-relic/03-monitor-new-relic.md [hunk 1]
```diff
@@ -45,11 +45,19 @@ For more information on using New Relic&reg;'s features, we encourage you to rev
...
+New Relic provides an availability monitoring service within their Synthetics tool suite which can be enabled by [contacting support](/guides/support/contact-support/). This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.
 
-Pantheon can provide New Relic Ping monitoring for free as part of the service. However, more advanced monitoring, including full browser testing and scripted interactions, is only available to customers on an annual contract and requires an additional cost. Contact our [sales team](https://pantheon.io/contact-us) or your dedicated account manager for more information.
+Pantheon can provide New Relic Synthetics ping monitoring as part of the service with a usage quota limited to 10K Synthetics checks per month for each Pantheon workspace.
```
Comment (stevector) [inline source/content/guides/new-relic/03-monitor-new-relic.md:52]:
```suggestion
Pantheon caps usage of ping monitoring at 10K Synthetics checks per month for each Pantheon workspace.
```

File: source/content/guides/new-relic/03-monitor-new-relic.md [hunk 2]
```diff
@@ -45,11 +45,19 @@ For more information on using New Relic&reg;'s features, we encourage you to rev
...
+
+<Alert title="Note" type="info">
+
+When a Pantheon workspace has more than 10K Synthetics checks are running monthly Pantheon will adjust the frequency or number of locations a Synthetics check originates from the reduce the number of checks. This is required to allow Synthetic to continue to operate correctly under the overall New Relic quota for Pantheon.
```
Comment (stevector) [inline source/content/guides/new-relic/03-monitor-new-relic.md:56]: @tyherbert will our customers know that/when this reduction happens?

File: source/content/guides/new-relic/03-monitor-new-relic.md (hunk 1 above)
Comment (tyherbert) [reply source/content/guides/new-relic/03-monitor-new-relic.md:52]: @stevector We need to clarify this: the quota doesn't include ping monitors (customers can use unlimited ping monitors); the limit only applies to non ping monitors. I'd propose something like this:
```suggestion
Pantheon provides unlimited New Relic Synthetics ping monitoring as part of the service. Still, there's a quota for all other monitor types (simple browser, scripted browser, and scripted API monitors). This quota is set at 10K Synthetics monthly checks for each Pantheon account, so it's crucial to manage it effectively.
```

File: source/content/guides/new-relic/03-monitor-new-relic.md (hunk 2 above)
Comment (tyherbert) [reply source/content/guides/new-relic/03-monitor-new-relic.md:56]: Yes, we plan to notify the customer via email if a change has been made to their Synthetics monitors.

**Comment by tyherbert** [comment]
@rachelwhitton Here is the PR for the release notes: https://github.com/pantheon-systems/documentation/pull/8980

File: source/content/guides/new-relic/03-monitor-new-relic.md (hunk 2 above)
Comment (stevector) [reply source/content/guides/new-relic/03-monitor-new-relic.md:56]: So is the flow something like
* Pantheon staff somehow sees that a workspace has exceed 10K
* Pantheon staff manually adjust settings in New Relic
* Pantheon staff manually emails customer?

Who from the workspace will get that email?

Comment (tyherbert) [reply source/content/guides/new-relic/03-monitor-new-relic.md:56]: Correct. This is something that Manoj and I will be working on. We are following the customer communication flow (https://docs.google.com/document/d/1Myagjy-2JhfnG7AqLs6clQ2YV0jxRvz_6nxQK898RQg/edit#heading=h.2licx9cbwmja). Currently we send an email to the site owner and cc the admins (this is a process that Shaq and Manoj put together).

**Pull request merged**